pages.it/common/vim.md

```diff
@@ -24,7 +24,7 @@
 
 `<Esc>/{{pattern}}<Invio>`
```

Comment:
Please change `pattern` to `regurlar_expresion` here (aswel) (in it's specified translation).